src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -922,8 +922,6 @@ ad_utility::streams::stream_generator ExportQueryExecutionTrees::
   result->logResultSize();
   LOG(DEBUG) << "Converting result IDs to their corresponding strings ..."
              << std::endl;
-  auto selectedColumnIndices =
-      qet.selectedVariablesToColumnIndices(selectClause, false);
```

Comment:
This is actual unused code I found while copy-pasting this code for reference. The same call that's actually being used can be found a couple of lines below